### ISSUE TITLE
API Only?

### ISSUE THREAD

**Issue by onedayitwillmake** [lml/commontator#159]
First, I'd like to say, great work on this gem! It's extremely easy to use, and versatile.

Are there any plans to make it work without any of the HTML mark up portions?  With minimal wrangling, I was able to invoke the methods via their respective routes and have them return JSON instead of `html` strings - but now I have to manage / patch the portions I overwrote as the library updates. 

As a user, and given that many new projects are using the new "API Only" mode of rails - Ideally there would be an "API" only option in Commontator to make it easier to leverage in such cases

**Comment by Dantemss** [comment]
What would be necessary for this?
I assume making most of the controller routes able to return JSON responses and that's it?

**Comment by onedayitwillmake** [comment]
Yeah that sounds about right, I had simply cloned the controllers and did just that and it pretty much worked.

That and some docs on the request params  and routes.

**Comment by martino-dot** [comment]
I agree that this should be added. This is also a feature that I was hoping would get added.
